engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/DebugLocalScope.java

```diff
@@ -188,15 +205,78 @@ boolean isMemberReadable(String memberName) {
   @ExportMessage
   @TruffleBoundary
   Object readMember(String member, @CachedLibrary("this") InteropLibrary interop)
-      throws UnknownIdentifierException {
+      throws UnknownIdentifierException, UnsupportedMessageException {
     if (!allBindings.containsKey(member)) {
       throw UnknownIdentifierException.create(member);
     }
     FramePointer framePtr = allBindings.get(member);
     var value = getValue(frame, framePtr);
+    if (value != null && isHostObject(value)) {
```

Comment:
`isHostObject` is probably not correct check. What about the same object coming from JavaScript? Or Python? They all should look the same, shouldn't they?